modules/gov/source/source.go

```diff
@@ -1,12 +1,15 @@
 package source
 
-import govtypes "github.com/cosmos/cosmos-sdk/x/gov/types/v1"
+import (
+	govtypesv1 "github.com/cosmos/cosmos-sdk/x/gov/types/v1"
+	govtypesv1beta1 "github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1"
+)
 
 type Source interface {
-	Proposal(height int64, id uint64) (*govtypes.Proposal, error)
-	ProposalDeposit(height int64, id uint64, depositor string) (*govtypes.Deposit, error)
-	TallyResult(height int64, proposalID uint64) (*govtypes.TallyResult, error)
-	DepositParams(height int64) (*govtypes.DepositParams, error)
-	VotingParams(height int64) (*govtypes.VotingParams, error)
-	TallyParams(height int64) (*govtypes.TallyParams, error)
+	Proposal(height int64, id uint64) (govtypesv1beta1.Proposal, error)
+	ProposalDeposit(height int64, id uint64, depositor string) (*govtypesv1.Deposit, error)
+	TallyResult(height int64, proposalID uint64) (*govtypesv1.TallyResult, error)
+	DepositParams(height int64) (*govtypesv1.DepositParams, error)
```

Comment:
Both types needed?

Reply:
Looks like it - it was a part of [this commit](https://github.com/forbole/bdjuno/commit/901a355b0b5e2d195f65936c8743ea489ee18766).